gamersnet_frontend/src/components/displayPosts/index.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import {withRouter} from 'react-router-dom';
+
+import APIFetch from '../../utilities/api';
+import Post from './post'
+
+class DisplayPosts extends React.Component{
+    _isMounted = false;
```

Comment:
This seems dangerous to do (modifying the abstracted React component lifecycle), is this safe?

Reply:
You may be right, and I'll try to update it like yours from the account_feature branch